assembl/indexing/changes.py

```diff
@@ -193,4 +194,19 @@ def tpc_abort(self, transaction):
         self._clear()
 
 
-changes = ElasticChanges(transaction.manager)
+_changes = None
+
+
+def get_changes():
+    global _changes
```

Comment:
It is invoked downstream, and it is initialized too late in process (dependent on config info) to be made a module variable.